.github/workflows/2.b-build-nifi-workbench-docker.yml

```diff
@@ -10,12 +10,15 @@ on:
 env:
   REGISTRY: ghcr.io
   NIFI_DOCKER_IMAGE_VERSION: 1.20.0-jdk18
-  LDES_NIFI_DOCKER_IMAGE_NAME: ghcr.io/informatievlaanderen/ldes-workbench-nifi
+  LDES_NIFI_DOCKER_IMAGE_NAME: ghcr.io/informatievlaanderen/ldi-workbench-nifi
 
 jobs:
   job-build-docker-workbench-nifi:
     name: Test build LDES Workbench NiFi Docker image
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
I know that I have added this myself when I was trying something but I think contents: read would work as well

Reply:
indeed. based on the NiFi repo workflows, I think read should also work